test/cases/filename/webpack.config.js

```diff
@@ -0,0 +1,24 @@
+const Self = require('../../../');
+
+module.exports = {
+  entry: {
+    'demo/js/main': './index.js',
```

Comment:
Should fix your test fail
```suggestion
    'expected/demo/js/main': './index.js',
```